src/main.cpp

```diff
@@ -1,9 +1,8 @@
 #include <iostream>
+#include <gtest/gtest.h>
 
-using namespace std;
-
-int main()
-{
-	cout << "Hello World!" << endl;
-	return 0;	
+int main(int argc, char** argv) {
+    testing::InitGoogleTest(&argc, argv);
```

Comment:
The unit tests should have their own `main.cpp` file, otherwise there will no place for the main application.